src/egit.h

```diff
@@ -1,4 +1,5 @@
 #include "emacs-module.h"
+#include "git2.h"
```

Comment:
Without it I get errors about unknown types for `git_commit` and `git_reference`

Reply:
That's because you forgot to include `git2.h` in `egit-branch.c`. But that's fine, you can leave this since `git_repository` is a type that is referred to in this file, actually.